src/main/java/com/contrastsecurity/utils/MetadataDeserializer.java

```diff
@@ -0,0 +1,48 @@
+package com.contrastsecurity.utils;
```

Comment:
This is a pretty good size piece of code. Can you add a few tests around it?

Reply:
Added test cases for Metadata deserializer.